.github/workflows/unit_tests.yml

```diff
@@ -23,7 +23,7 @@ jobs:
   unit_tests:
     strategy:
       matrix:
-        python-version: [ 3.6, 3.7, 3.8, 3.9 ]
+        python-version: [ 3.7, 3.8, 3.9 ]
```

Comment:
It's strange that python 3.6 tests failed because of huggingface
[Here](https://github.com/NeonGeckoCom/neon-tts-plugin-coqui-ai/runs/6192794812?check_suite_focus=true) it's passed. But it depends on situation

Reply:
That is interesting... Not sure why, a cursory search indicated huggingface was importing a lib included in 3.7+